app/services/PaymentService.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import models.Period
+import models.requests.{PaymentPeriod, PaymentRequest}
+import uk.gov.hmrc.domain.Vrn
+
+import javax.inject.Inject
+
+class PaymentService @Inject()() {
```

Comment:
Given how simple this is (i.e. it doesn't need any other services etc. injected in), could it just be an apply method on the PaymentRequest companion object?

Reply:
Yep, will move it over and delete the service 👍🏻